File: Kids with Greatest number of Candies/Sol.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies)
    {
        int max = *max_element(candies.begin(), candies.end());
        vector<bool> res;
        for (int i = 0; i < candies.size(); i++)
        {
            candies[i] + extraCandies >= max ? res.push_back(true) : res.push_back(false);
        }
        return res;
    }
};

File: Find Anagram/Sol.java
class Solution {
    public int[] anagramMappings(int[] A, int[] B)
    {
        int[] res = new int[A.length];
        for (int i = 0; i < A.length; i++)
        {
            for (int j = 0; j < B.length; j++)
            {
                if (A[i] == B[j])
                {
                    res[i] = j;
                }
            }
        }
        return res;
    }
}
